e2e/cypress/tests/integration/enterprise/cloud/billing/self_hosted_pricing_modal_spec.js

```diff
@@ -169,4 +169,22 @@ describe('Self hosted Pricing modal', () => {
         cy.get('#enterprise').should('be.visible');
         cy.get('#start_trial_btn').should('not.be.disabled');
     });
+
+    it('Upgrade button should open air gapped modal when hosted signup is not availabe', () => {
+        cy.apiAdminLogin();
+
+        cy.intercept({method: 'GET', url: '/signup_available'}).as('airGappedCheck')
```

Comment:
I don't cypress often enough to be confident, but I think this means the request still gets made to the server. I think we want to add a third argument which is the [stub response](https://docs.cypress.io/api/commands/intercept#Usage) (the example with 3 arguments). Doing it that way means the network request doesn't really happen, which means we don't have to worry about mattermost-server being configured a certain way, or worry about a running instance of CWS for it to talk to. Sometimes we *do* want all that heavy set up, but for this test I think its acceptable to stub out the response.